python/lsst/daf/butler/core/butlerConfig.py

```diff
@@ -58,6 +58,8 @@ class ButlerConfig(Config):
 
     def __init__(self, other=None):
 
+        self.configDir = None
+
```

Comment:
Maybe move this down to just above the block that overrides it, to keep the logic that initializes this attribute in one place?

Reply:
Turned out that I have to leave it there because there was a bug in the original version of this code that meant that configDir was lost if a ButlerConfig was seeded from another ButlerConfig.